web_src/js/features/imagediff.js

```diff
@@ -98,174 +101,184 @@ export function initImageDiff() {
-          info.$images.each(function() {
-            this.setAttribute('width', bounds.width);
-            this.setAttribute('height', bounds.height);
-          });
+          for (const el of info.images) {
+            el.setAttribute('width', bounds.width);
+            el.setAttribute('height', bounds.height);
+          }
           hideElem(info.boundsInfo);
         }
       }
     }));
 
-    const $imagesAfter = imageInfos[0].$images;
-    const $imagesBefore = imageInfos[1].$images;
+    const imagesAfter = imageInfos[0].images;
+    const imagesBefore = imageInfos[1].images;
 
-    initSideBySide(this, createContext($imagesAfter[0], $imagesBefore[0]));
-    if ($imagesAfter.length > 0 && $imagesBefore.length > 0) {
-      initSwipe(createContext($imagesAfter[1], $imagesBefore[1]));
-      initOverlay(createContext($imagesAfter[2], $imagesBefore[2]));
+    this.initSideBySide(createContext(imagesAfter[0], imagesBefore[0]));
+    if (imagesAfter.length > 0 && imagesBefore.length > 0) {
+      this.initSwipe(createContext(imagesAfter[1], imagesBefore[1]));
+      this.initOverlay(createContext(imagesAfter[2], imagesBefore[2]));
     }
+    queryElemChildren(containerEl, '.image-diff-tabs', (el) => el.classList.remove('is-loading'));
+  }
 
-    queryElemChildren(this, '.image-diff-tabs', (el) => el.classList.remove('is-loading'));
-
-    function initSideBySide(container, sizes) {
-      let factor = 1;
-      if (sizes.maxSize.width > (diffContainerWidth - 24) / 2) {
-        factor = (diffContainerWidth - 24) / 2 / sizes.maxSize.width;
-      }
+  initSideBySide(sizes) {
+    let factor = 1;
+    if (sizes.maxSize.width > (this.diffContainerWidth - 24) / 2) {
+      factor = (this.diffContainerWidth - 24) / 2 / sizes.maxSize.width;
+    }
 
-      const widthChanged = sizes.imageAfter && sizes.imageBefore && sizes.imageAfter.naturalWidth !== sizes.imageBefore.naturalWidth;
-      const heightChanged = sizes.imageAfter && sizes.imageBefore && sizes.imageAfter.naturalHeight !== sizes.imageBefore.naturalHeight;
-      if (sizes.imageAfter) {
-        const boundsInfoAfterWidth = container.querySelector('.bounds-info-after .bounds-info-width');
-        if (boundsInfoAfterWidth) {
-          boundsInfoAfterWidth.textContent = `${sizes.imageAfter.naturalWidth}px`;
-          boundsInfoAfterWidth.classList.toggle('green', widthChanged);
-        }
-        const boundsInfoAfterHeight = container.querySelector('.bounds-info-after .bounds-info-height');
-        if (boundsInfoAfterHeight) {
-          boundsInfoAfterHeight.textContent = `${sizes.imageAfter.naturalHeight}px`;
-          boundsInfoAfterHeight.classList.toggle('green', heightChanged);
-        }
+    const widthChanged = sizes.imageAfter && sizes.imageBefore && sizes.imageAfter.naturalWidth !== sizes.imageBefore.naturalWidth;
+    const heightChanged = sizes.imageAfter && sizes.imageBefore && sizes.imageAfter.naturalHeight !== sizes.imageBefore.naturalHeight;
```

Comment:
Can these two lines be simplified to `sizes.imageAfter?.naturalWidth !== sizes.imageBefore?.naturalWidth`?

Reply:
No, the meaning is different.